### ISSUE TITLE
Request: allow to export to vectordrawable too

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [romannurik/AndroidAssetStudio#107]
because now it's supported on various Android versions.

**Comment by romannurik** [comment]
that's a good idea, but will take a bunch of new code... i hope to revamp the asset studio at some point, this would be a great feature request for that time

**Comment by AndroidDeveloperLB** [comment]
You could export to SVG, so that designers could also modify the files easier.

**Comment by romannurik** [comment]
Explored this; it's basically not feasible to export vectors for any foreground images except clipart. I think it's probably better to use the [Android Icon Animator](https://romannurik.github.io/AndroidIconAnimator/) tool.

See also #121 and #123

**Comment by AndroidDeveloperLB** [comment]
Why not ?
You can already convert SVG files to VectorDrawables, like on Android Studio.
When you create the image files from the selection (not from image files of the user), it uses SVG files too, no?

**Comment by romannurik** [comment]
Yes it would be possible for clipart, but not custom uploaded images, or text. I'll actually re-open and re-frame the request.

**Comment by AndroidDeveloperLB** [comment]
Text is not possible only because VectorDrawable doesn't support it (sadly), right? 
Just like it doesn't support many other SVG features, like gradients...
But can you allow to export to SVG (not VectorDrawbles) the text types ?

**Comment by romannurik** [comment]
text isn't possible because we'd need to convert the text to paths, and this isn't possible in the browser AFAIK (you can't extract shapes from a web font).

**Comment by AndroidDeveloperLB** [comment]
But fonts are by definition vectorized, no? So that no matter how you resize them, they will keep being sharp and nice...
There are even tutorials for this. Here's an example:
http://www.graphic-design-employment.com/adobe-illustrator-how-to-convert.html

**Comment by romannurik** [comment]
I'm not sure I follow. For a web-based tool to generator vector icons from text that uses a web font, the web-based tool needs to somehow have access to the font's path data. This isn't feasible right now. If you really want a vector drawable where the shape is a letterform from a web font, the best option is to download the font, use it in something like Illustrator or Sketch, convert the type layer to outlines ('expand' in Illustrator), export to SVG, and convert to vector drawables using any one of the popular SVG-to-VD converters (e.g. the Vector Asset Studio in Android Studio, or the Android Icon Animator tool, or something else)

**Comment by AndroidDeveloperLB** [comment]
It's not possible to get the font's paths? So how does it get drawn? Only by black-box commands of the web browser's engine?

**Comment by romannurik** [comment]
Yes take a look at `imagefield.js` if you're interested.

**Comment by AndroidDeveloperLB** [comment]
OK sorry. Just not my field, so I sometimes don't know what is possible and what's not on a web browser.
It's too bad though. Isn't it even possible to handle fonts that are inside the website? Like Roboto ?
